simple_ado/wiki.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
```

Comment:
I think a few older pages have this, but it's not necessary.

Reply:
True, I must've copied from an old file. Updated 👍